AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/AppLovin/Rewarded/Basic Integration/ALDemoRewardedVideosViewController.m

```diff
@@ -62,17 +62,10 @@ - (void)rewardValidationRequestForAd:(ALAd *)ad didSucceedWithResponse:(NSDictio
     /* AppLovin servers validated the reward. Refresh user balance from your server.  We will also pass the number of coins
      awarded and the name of the currency.  However, ideally, you should verify this with your server before granting it. */
     
-    // i.e. - "Coins", "Gold", whatever you set in the dashboard.
-    NSString *currencyName = response[@"currency"];
-    
-    // For example, "5" or "5.00" if you've specified an amount in the UI.
-    NSString *amountGivenString = response[@"amount"];
-    NSNumber *amountGiven = @([amountGivenString floatValue]);
```

Comment:
Can we keep this in there? This is primarily to demo to the pub on how to retrieve currency and amount from our reward callback. Instead of deleting it to silence the warning I would probably log it instead.